drake_cmake_installed/README.md

```diff
@@ -72,3 +72,35 @@ Drake specific Examples:
 Compatibility Examples:
 
 * [PCL](src/pcl/README.md)
+
+# Developer Testing
+
+If you are a Drake Developer making build or API changes that may affect the
+downstream interface, please test this locally on your system.
+
+These build instructions are adapted from those above, but will use an existing
+source tree of Drake (but *not* installing it to `/opt/drake`),
+build this project, and then run all available tests:
+
+```shell
+# Build development version of Drake, ensuring no old artifacts are present.
+cd drake  # Where you are developing.
+rm -rf ../drake-build && mkdir ../drake-build && cd ../drake-build
+cmake ../drake  # Configure Gurobi, Mosek, etc, if needed.
+# Build locally.
+make -j4
```

Comment:
Just `make` is fine. The `-j4` is redundant since everything is proxied through Bazel.